app/views/repairs/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1 class='repair-header d-flex a-center j-center'>New Repair</h1>
+<div class= 'add-equipment add-repair d-flex f-col a-center j-center'> 
+    <%=form_with url: repairs_path, scope: :data do |f| %>
+      <div class="field3 d-flex a-center j-center f-col" >
+        <%= f.label :equipment_id, 'Select Equipment' %>
+         <%= f.select :equipment_id, @equipments.collect { |equipment| [equipment.name, equipment.id] }, class: "full-width", :prompt => 'Select an Equipment', :required=>true %>
+      </div>
+      <div class="field3 d-flex a-center j-center f-col">
+        <%= f.label :name, 'Name'  %>
+        <%= f.text_field :name, :required=>true %>
+      </div>
+       <div class="field3 d-flex a-center j-center f-col">
+        <%= f.label :amount, 'Amount'  %>
+        <%= f.number_field :amount, :required=>true %>
+      </div>
+    <div class="action1 d-flex a-center f-col">
+         <%= f.submit "Add Repair"%>
+    </div>
+      
+<% end %>
+</div>
```

Comment:
- [ ]  When the user clicks on the "Back" button (<), the user navigates to the home page, please implement the back button when it comes to the mobile version.

![image](https://user-images.githubusercontent.com/55635977/158817993-af61c134-87f1-4e45-aa05-7673561e7e8c.png)

Reply:
Well done!